content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,152 @@
+
+- **Authorization for End-User Groups**. Allows authorization based on `groups` claim or any list-typed claims in JWT.
+See [Authorization for groups and list claims](/docs/tasks/security/rbac-groups/) for more information.
+
+- **End-User Authentication with Per-Path Requirements**. Allows you to enable or disable JWT authentication based on the request path.
+See [End-user authentication with per-path requirements](/docs/tasks/security/authn-policy/#end-user-authentication-with-per-path-requirements) for 
+more information.
+
+- **External Certificate Management on Ingress Gateway Controller**. Dynamically loads and rotates external certificates.
+
+- **Vault PKI Integration**. Provides stronger security with Vault-protected signing keys and facilitates integration with existing Vault PKIs.
+See [Istio Vault CA Integration](/docs/tasks/security/vault-ca) for more information.
+
+- **Customized (non `cluster.local`) Trust Domains**. Supports organization- or cluster-specific trust domains in the identities.
+
+## Multiclusters
+
+- **Non-Routable L3 Networks**. Enabled using a single Istio control plane in multicluster environments with non-routable
+L3 networks.
+TBD: LINK
+
+- **Multiple Control Planes**. Added support of multiple Istio control planes for multicluster environments.
+TBD: LINK
+
+## Policies and telemetry
+
+- **Policy Checks Off By Default**. Changed policy checks to be turned off by default which improves performance for most customer scenarios.
+TBD: LINK
+
+- **Kiali**. Replaced the [Service Graph addon](https://github.com/istio/istio/issues/9066) with [Kiali](https://www.kiali.io) to provide
+a richer visualization experience. See the [Kiali task](/docs/tasks/telemetry/kiali/) for more details.
+
+- **Reduced Overhead**. Added several performance and scale improvements including:
+
+    - Significant reduction in default collection of Envoy-generated statistics (configurable via annotations TBD: LINK)
+
+    - Added load-shedding functionality to Mixer workloads.
+
+    - Improved the protocol between Envoy and Mixer.
+
+- **Control Headers and Routing**. Added the option to create adapters to influence
+an incoming request's headers and routing. TBD: LINK
+
+- **Out of Process Adapters**. The out-of-process adapter functionality is now ready for production
+TBD: LINK
+ 
+- **Tracing Improvements**. There have been many improvements in our overall tracing story:
+
+    -  You can now use 128 bit trace IDs.
+
+    -  Added some enhanced deployment options for [Jaeger](https://istio.io/docs/tasks/telemetry/distributed-tracing/jaeger/)
+    and [Zipkin](https://istio.io/docs/tasks/telemetry/distributed-tracing/zipkin/). TBD: are these right links for "deployment options"?
+
+    - Added support of sending trace data to [LightStep](https://preliminary.istio.io/docs/tasks/telemetry/distributed-tracing/lightstep/)
+
+    - Added the option to disable tracing for Mixer-backed services entirely. TBD: ADD LINK
```

Comment:
Relevant PR: https://github.com/istio/istio/pull/9615

Reply:
Docs please, customers shouldn't be sent to PRs.

Reply:
There isn't a specific doc for just turning off tracing at the Mixer level.  So, I'd drop the TBD for a link then.

Reply:
Let's remove the link. I don't think this needs a doc (covered in FAQs)

Reply:
Done